qiskit/dagcircuit/_dagcircuit.py

```diff
@@ -1341,6 +1352,30 @@ def property_summary(self):
                    "operations": self.count_ops()}
         return summary
 
+    def add_dag_at_the_end(self, dag, layout=None):
+        """Add `dag` at the end of `self`, using layout for wireing.
+        If `layout` is not provided, creates a layout where wires are
+        connected one-to-one."""
+        for qreg in dag.qregs.values():
+            with suppress(DAGCircuitError):
+                self.add_qreg(QuantumRegister(qreg.size, qreg.name))
```

Comment:
Why do we need this suppress here? Can't you just do ```if qreg not in self.qregs```?

Reply:
good point. Fixed in b27c2b1.